tests/lib/engine-dummy/Cargo.toml

```diff
@@ -2,9 +2,9 @@
 name = "wasmer-engine-dummy"
 version = "1.0.0-alpha01.0"
 authors = ["Wasmer Engineering Team <engineering@wasmer.io>"]
-description = "wasmer testing utils"
+description = "Wasmer placeholder engine"
+license = "MIT"
 edition = "2018"
-publish = false
```

Comment:
Why publish engine-dummy?

Reply:
It's a dependency of `wasmer` (API) because it's used in Wasmer the CLI...

I agree that this was unfortunate, but I figured it was better to publish it now and refactor it later as I thought the refactor would be relatively large and that there'd be no way for me to publish it last week.


I agree that this is something we should file an issue for and investigate

Reply:
So to clarify, it's an accidental hard-requirement that it be published.  It may be possible to rework it so that we don't need to publish it in the future, but I chose to just publish it rather than try to fix it on Friday

Reply:
@MarkMcCaskey CLI doesn't use engine-dummy as far as I can tell?